fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json

```diff
@@ -1,98 +1,124 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://example.com/schemas/abstract",
```

Comment:
is this line required?

Reply:
No, this was just required for another jsonschema2md script. It is removed now.